src/common/marlin_server.hpp

```diff
@@ -0,0 +1,122 @@
+// marlin_server.hpp
+#pragma once
+
+#include "marlin_server.h"
+#include "client_response.hpp"
+
+/*****************************************************************************/
+//C++ only features
+
+//notyfy all clients to create finit statemachine, must match fsm_create_t signature
+void fsm_create(ClinetFSM type, uint8_t data);
+//notyfy all clients to destroy finit statemachine, must match fsm_destroy_t signature
+void fsm_destroy(ClinetFSM type);
+//notyfy all clients to change state of finit statemachine, must match fsm_change_t signature
```

Comment:
Typo notyfy -> notify. Put TODO for must match fsm_change_t signature comment.

Reply:
done